src/interfaces/token.ts

```diff
@@ -221,4 +258,16 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
       return result;
     }
   }
+
+  /**
+   * Tokens that can be used to zap into or out of a vault. Supplementary to those fetched from zapper's api
+   */
+  private zapTokenAddresses(): Address[] {
+    if (this.chainId === 250) {
```

Comment:
Fine for now  but we should find a way to do this such that we don't have do add conditional chain logic in lots of places